core/src/main/scala/akka/persistence/cassandra/EventsByTagMigration.scala

```diff
@@ -100,7 +100,7 @@ object EventsByTagMigration {
 class EventsByTagMigration(system: ActorSystem, pluginConfigPath: String = "akka.persistence.cassandra") {
   private[akka] val log = Logging.getLogger(system, getClass)
   private lazy val queries = PersistenceQuery(system).readJournalFor[CassandraReadJournal](pluginConfigPath + ".query")
-  private implicit val materialiser = ActorMaterializer()(system)
+  private implicit val sys: ActorSystem = system
```

Comment:
Implicit materializer is provided if there is an implicit `ActorSystem` available, so manually created materializers were replaced with implicitly defined actor systems.

Reply:
👍